doc/models/Load/FlexibleLoad/FeasibleDomain.md

```diff
@@ -584,6 +584,96 @@ Notice that when the voltages were too low or too high, the Euclidean projection
 reduction of the produced active power and the (produced and consumed) reactive power to ensure a
 feasible solution. Like before, there is one point per volt and several points overlap.
 
+### $Q^{\min}$ and $Q^{\max}$ limits
+
+It is also possible to define a minimum and maximum reactive power value. In that case, the feasible domain is
+constrained between those two values.
+
+```{image} /_static/Load/FlexibleLoad/Domain_Pconst_QU_Qmin_Qmax.svg
+:width: 300
+:align: center
+```
+
+```python
+import numpy as np
+
+from roseau.load_flow import Q_, FlexibleParameter, Control, Projection
+
+# Flexible parameter
+fp = FlexibleParameter(
+    control_p=Control.constant(),
+    control_q=Control.q_u(
+        u_min=Q_(210, "V"), u_down=Q_(220, "V"), u_up=Q_(240, "V"), u_max=Q_(250, "V")
+    ),
+    projection=Projection(type="euclidean"),
+    s_max=Q_(5, "kVA"),
+    q_min=Q_(-3, "kVAr"),  # <---- set Q_min >= -S_max
+    q_max=Q_(4, "kVAr"),  # <---- set Q_max <= S_max
+)
+
+# We want to get the res_flexible_powers for a set of voltages norms
+voltages = np.arange(205, 256, dtype=float)
+
+# and when the theoretical power is the following
+power = Q_(-2.5, "kVA")
+
+# Get the resulting flexible powers for the given theoretical power and voltages list.
+auth = ("username", "password")
+res_flexible_powers = fp.compute_powers(auth=auth, voltages=voltages, power=power)
+```
+
+The variable `res_flexible_powers` contains the powers that have been really produced by
```

Comment:
```suggestion
The variable `res_flexible_powers` contains the powers that have been actually produced by
```
There is another occurrence in this file that I forgot to fix, maybe you can fix it as well.